src/cordial_sound/scripts/sound_listener.py

```diff
@@ -29,10 +29,8 @@ def play_sound(data):
 
     p = pyaudio.PyAudio()
 
-    """
```

Comment:
I don't think this change is necessary - it's good to keep the diff small so it's easy to see what has happened.

Reply:
This was showing up as an issue in Codefactor. That's why I had to change it